Cubical/Categories/Presheaves.agda

```diff
@@ -0,0 +1,76 @@
+{-# OPTIONS --cubical --postfix-projections #-}
+
+module Cubical.Categories.Presheaves where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.Isomorphism
+open import Cubical.Foundations.Equiv
+open import Cubical.HITs.PropositionalTruncation
+
+open import Cubical.Categories.Category
+open import Cubical.Categories.Functor
+open import Cubical.Categories.NaturalTransformation
+open import Cubical.Categories.Sets
+
+module _ (ℓ : Level) where
+  PSH : Precategory ℓ → Precategory (ℓ-suc ℓ)
+  PSH 𝒞 = FUNCTOR (𝒞 ^op) (SET ℓ)
```

Comment:
this resulting category `isCategory`, as `SET` `isCategory`, isn't it so? (probably useful result to add to `NaturalTransformations`). If i'm not mistaken, that's HoTT theorem 9.2.5

Reply:
Yes it is, but I think it can be added to a future PR